README.md

```diff
@@ -50,42 +50,7 @@ pip install astro-sdk-python[amazon,google,snowflake,postgres]
 
 1. Copy the following DAG into a file named `calculate_popular_movies.py` and add it to the `dags` directory of your Airflow project:
```

Comment:
I'd actually prefer if we kept the DAG code visible in the README. Even though it's longer, the reader won't have to click into another page/ run a command before they see how the SDK works. We can still keep the other DAG file, but this would serve as the window into what the SDK can do

Reply:
If we could embed the code it would be the best scenario from both perspectives